frontend/pages/[lng]/auth/client-details.tsx

```diff
@@ -4,6 +4,8 @@ import axios from "axios"
 import styled from "@emotion/styled"
 import { getAccessToken } from "../../../lib/authentication"
 
+const BASE_URL = "https://mooc.fi"
```

Comment:
Maybe use a production/dev switch with these, so that they work locally as well, ie. `process.env.NODE_ENV === 'production' ? "https://mooc.fi" : "http://localhost:4000"`